app/errors.js

```diff
@@ -3,8 +3,16 @@ const internalError = (message, internalCode) => ({
   internalCode
 });
 
+const externalError = (message, externalCode) => ({
+  message,
+  externalCode
+});
```

Comment:
This ends up being the same as internalError but with different names. Id just reuse internalError, after all `api_error` already tells us the error comes from an external source.